### PR TITLE
fix: homogenize all displayed error messages style

### PR DESCRIPTION
**Pull request by adriguy**
https://app.clickup.com/t/4ygzep

## Pull Request checklist:

- [x] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [ ] Create automatic tests
- [x] No automatic tests failures
- [x] Test manually the implemented changes
- [x] Review my own code (indentation, syntax, style, simplicity, readability)
- [x] Wonder if you can improve the existing code

### CODE REVIEW

File: src/commands/branch.js [hunk 1]
```diff
@@ -19,7 +20,8 @@ class BranchCommand extends AbstractAuthenticatedCommand {
     } catch (error) {
       const customError = BranchManager.handleBranchError(error);
 
-      return this.error(customError);
+      logger.error(customError);
+      return this.exit(2);
```
Comment (larcin) [inline src/commands/branch.js:24]: [Misc] Why did you make an `exit(2)` here?

Comment (larcin) [reply src/commands/branch.js:24]: Same question for all `exit(2)` across this file

File: src/commands/environments/copy-layout.js [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
 const AbstractAuthenticatedCommand = require('../../abstract-authenticated-command');
+const EnvironmentManager = require('../../services/environment-manager');
 const withCurrentProject = require('../../services/with-current-project');
+const logger = require('../../services/logger');
 const envConfig = require('../../config');
 
 class CopyLayoutCommand extends AbstractAuthenticatedCommand {
   async runIfAuthenticated() {
-    const logError = this.error.bind(this);
+    const oclifExit = this.exit.bind(this);
```
Comment (larcin) [inline src/commands/environments/copy-layout.js:12]: 👏

File: src/commands/environments/create.js [hunk 3]
```diff
@@ -19,11 +20,8 @@ class CreateCommand extends AbstractAuthenticatedCommand {
         const errorData = JSON.parse(error.response.text);
         if (errorData && errorData.errors && errorData.errors.length
           && errorData.errors[0] && errorData.errors[0].detail) {
-          this.error(errorData.errors[0].detail, {
-            code: errorData.errors[0].status,
-            exit: 1,
-          });
-          return;
+          logger.error(errorData.errors[0].detail);
+          this.exit(1);
```
Comment (larcin) [inline src/commands/environments/create.js:24]: [MISC] If you want to stay iso with what existed (and avoid the following `throw error`, you should return here.

File: src/services/job-state-checker.js [hunk 4]
```diff
@@ -5,10 +5,11 @@ const { promisify } = require('util');
 const jobDeserializer = require('../deserializers/job');
 const config = require('../config');
 const { getAuthToken } = require('./authenticator');
+const logger = require('../services/logger');
```
Comment (larcin) [inline src/services/job-state-checker.js:8]:
```suggestion
const logger = require('./logger');
```

File: src/commands/branch.js (hunk 1 above)
Comment (adriguy) [reply src/commands/branch.js:24]: As discussed, it is the default exit code thrown by `this.error`...
So to stay iso and satisfy the current tests, I had to specify the exit 2.

File: src/commands/environments/create.js (hunk 3 above)
Comment (adriguy) [reply src/commands/environments/create.js:24]: As discussed, after the `this.exit(1)`, all code added is flagged as unreachable by my IDE.
So let's keep it that way!

**Pull request merged**